package.json

```diff
@@ -25,7 +25,8 @@
     "postinstall": "node scripts/npm/postinstall",
     "release": "./scripts/travis/release.sh",
     "build": "gulp build",
-    "test": "gulp test && node test/accessibility/wcag.js"
+    "test": "gulp test && node test/accessibility/wcag.js",
+    "cf-link": "cd ./src/cf-core && npm link && cd ../cf-buttons/ && npm link"
```

Comment:
This will definitely be unruly and is going to require some longer term thinking. Maybe we can hook into a manifest and loop these?